README.md

```diff
@@ -64,3 +65,9 @@ yarn run format:prettier
 For most of the tools, the configuration is in the `package.json` to reduce the amount of files in your project.
 
 If you customize the configuration a lot, you can consider moving them to individual files.
+
+## Deploying using `gh-pages`
+
+```
+yarn run deploy -e <review_app_name>
+```
```

Comment:
Please add 

Review app URL will be
[https://internetarchive.github.io/iaux-collection-browser/<review app name>(https://internetarchive.github.io/iaux-collection-browser/)